react/error.tsx

```diff
@@ -17,25 +17,51 @@ class ErrorPage extends Component {
   public state = { enabled: false }
 
   public componentDidMount() {
-    window.setTimeout(() => {
-      this.setState({ enabled: true })
-    }, 5000)
+    try {
+      const error = window?.__ERROR__
+      const requestId = window?.__REQUEST_ID__
+      const defaultError =
+        'Render Runtime renderered an error page and there is no error or request id available'
+
+      const runtime = window?.global?.__RUNTIME__ ?? {}
+
+      const {
+        account = 'N/A',
+        workspace = 'N/A',
+        culture: { locale } = { locale: 'N/A' },
+        route: { path } = { path: 'N/A' },
+        loadedDevices = ['N/A'],
+        production = 'N/A',
```

Comment:
_["The null value represents the intentional absence of any object value."](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/null)_ :hugs: 

```suggestion
        account = 'null,
        workspace = null,
        culture: { locale } = { locale: null },
        route: { path } = { path: null },
        loadedDevices = [null],
        production = null,
```

Reply:
Just a suggestion! `"N/A"` LGTM

Reply:
good catch! I agree

Reply:
I would  also suggest extracting the creation of `errorInfo` to private method. This is purely to decrease the size of the method and maintain readability.
Code LGTM, just a suggestion, too

```ts
  private extractErrorInfo(runtime: any): any {
    const {
      account = null,
      workspace = null,
      culture: { locale } = { locale: null },
      route: { path } = { path: null },
      loadedDevices = [null],
      production = null,
    } = runtime

    const errorInfo = {
      account,
      workspace,
      locale,
      path,
      device: loadedDevices[0],
      production,
    }

    return errorInfo
  }
  ```
  
  And call it when we add the extras to the current scope
   ```ts
      const errorInfo = this.extractErrorInfo(runtime);
      getCurrentScope().setExtras(errorInfo)
   ```